### PR TITLE
doc(provider): fix shared config file parameter issue

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.796s
```

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -112,8 +112,8 @@ Usage:
 
 ```terraform
 provider "huaweicloud" {
-  shared_configuration_file = "/home/tf_user/.hcloud/config.json"
-  profile                   = "customprofile"
+  shared_config_file = "/home/tf_user/.hcloud/config.json"
+  profile            = "customprofile"
```
Comment (hmbig2) [inline docs/index.md:116]: s/customprofile/activeProfile

Comment (hmbig2) [inline docs/index.md:116]: s/customprofile/activeProfile

Comment (niuzhenguo) [reply docs/index.md:116]: customeprofile is more common as other cloud providers

**Pull request merged**